spec/features/user_views_index_page_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+feature 'user ', %(
+As a use, when I visit the root page
+I want to see a list of shows to review.
+So that I can decide which one to judge.
+
+  Acceptance Criteria
+  [ ] - I must see the name of each show
+
+) do
+
+  scenario 'user views all shows' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.